diff --git a/lib/neutron b/lib/neutron
index e8f26ac..e0fd855 100644
--- a/lib/neutron
+++ b/lib/neutron
@@ -526,7 +526,7 @@ function create_neutron_initial_network {
                         done
                     fi
                     sudo ip -6 route add $IPV6_PRIVATE_RANGE via $IPV6_ROUTER_GW_IP dev $PUBLIC_BRIDGE
-                    sudo ip netns exec qrouter-$ROUTER_ID ip -6 route add ::/0 via $IPV6_EXT_GW_IP dev qg-${IPV6_ROUTER_GW_PORT:0:11}
+		    #sudo ip netns exec qrouter-$ROUTER_ID ip -6 route add ::/0 via $IPV6_EXT_GW_IP dev qg-${IPV6_ROUTER_GW_PORT:0:11}
                 fi
                 if [[ "$Q_USE_NAMESPACE" == "False" ]]; then
                     # Explicitly set router id in l3 agent configuration
